tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -180,14 +192,17 @@ func (a *AwsProvider) CreateWindowsVM() (windowsVM types.WindowsVM, err error) {
 		log.Printf("failed to assign name for instance: %s, %v", instanceID, err)
 	}
 
-	// Get the public IP
-	publicIPAddress, err := a.GetPublicIP(instanceID)
-	if err != nil {
-		return nil, err
+	var publicIPAddress, privateIPAddress string
```

Comment:
I maybe missing something, but is there a reason not to use short hand declaration in local scope?

Reply:
I understand declaring `publicIPAddress` here but why do you need to declare `privateIPAddress`?